recipes/ann/build.sh

```diff
@@ -0,0 +1,20 @@
+if [ "$(uname)" == "Darwin" ];
+then
+    export CC=clang
+    export CXX=clang++
```

Comment:
Could you please remove those ``export CC`` and ``export CXX`` ?

Reply:
Yup, just removed them